PostFreeSurfer/PostFreesurferPipelineLongPrep.sh

```diff
@@ -0,0 +1,418 @@
+#!/bin/bash
+
+# # PostFreeSurferPipelineLongPrep.sh
```

Comment:
The filename itself should capitalize `Surfer` too, for consistency.

Reply:
ok